sessions/05-R-estimation-and-the-renewal-equation.qmd

```diff
@@ -3,3 +3,175 @@ title: "R estimation and the renewal equation"
 order: 5
 ---
 
+```{r echo = FALSE}
+set.seed(123)
+```
+
+# Objectives
+
+The aim of this session is to introduce the renewal equation as an infection generating process, and to show how it can be used to estimate a time-varying reproduction number.
+
+# Libraries used
+
+In this session we will use the `nfidd` package to load the data set of infection times, the `dplyr` and `tidyr` packages for data wrangling, `ggplot2` library for plotting, the `here` library to find the stan model, and the `cmdstanr` library for using stan.
+We will also use the `tidybayes` package for extracting results of the inference.
+
+```{r libraries, message = FALSE}
+library("nfidd")
+library("dplyr")
+library("tidyr")
+library("ggplot2")
+library("here")
+library("cmdstanr")
+library("tidybayes")
+```
+
+::: {.callout-tip}
+The code in this session can be run as an interactive notebook using RStudio, or copied-and-pasted into an R session.
+It needs to be run inside the course repository so that the `here()` commands below find the stan model files.
+:::
+
+# The renewal equation as a process model for infectious diseases
+
+This session represents a shift from the models used in the previous session.
```

Comment:
I think this could be linked instead to the previous convolution session where we had to have a vague `Normal(0, 10)` model of infections. The renewal process is then justified as being closer to the "real" infection process and so  should help the model recover more realistic infections - especially in settings where any delays are also unknown/noisy... 

We could then discuss how this Rt is specifically also useful and often a target for inference in and of itself

Reply:
good idea - have updated text